### PR TITLE
Fixed broken build and made some additional L10n corrections to the layout.

### PR DESCRIPTION
**Pull request by kc284**

### CODE REVIEW

File: XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.ja.resx [hunk 1]
```diff
@@ -493,16 +490,16 @@
...
   </data>
   <data name="pictureBox1.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
-    <value>Top, Bottom, Left</value>
+    <value>Left</value>
   </data>
   <data name="pictureBox1.ImeMode" type="System.Windows.Forms.ImeMode, System.Windows.Forms">
     <value>NoControl</value>
   </data>
   <data name="pictureBox1.Location" type="System.Drawing.Point, System.Drawing">
-    <value>3, 3, 0, 0</value>
```
Comment (kc284) [inline XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.ja.resx:502]: This here was breaking the build.

File: XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.zh-CN.resx [hunk 2]
```diff
@@ -247,7 +247,7 @@
     <value>3, 378</value>
   </data>
   <data name="progressBar1.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>3, 64</value>
```
Comment (kc284) [inline XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.zh-CN.resx:250]: This was also breaking the build.

File: XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.zh-CN.resx [hunk 3]
```diff
@@ -499,10 +496,10 @@
     <value>NoControl</value>
   </data>
   <data name="pictureBox1.Location" type="System.Drawing.Point, System.Drawing">
-    <value>3, 3, 0, 0</value>
```
Comment (kc284) [inline XenAdmin/Wizards/DRWizards/DRFailoverWizardPrecheckPage.zh-CN.resx:502]: This was also breaking the build.

File: XenAdmin/Wizards/DRWizards/DRFailoverWizardRecoverPage.ja.resx [hunk 4]
```diff
@@ -274,7 +274,7 @@
     <value>468, 305</value>
   </data>
   <data name="linkLabel1.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>3, 315</value>
```
Comment (kc284) [inline XenAdmin/Wizards/DRWizards/DRFailoverWizardRecoverPage.ja.resx:277]: This was also breaking the build.

File: XenAdmin/Wizards/DRWizards/DRFailoverWizardRecoverPage.zh-CN.resx [hunk 5]
```diff
@@ -274,7 +274,7 @@
     <value>468, 305</value>
   </data>
   <data name="linkLabel1.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
-    <value>3, 315</value>
```
Comment (kc284) [inline XenAdmin/Wizards/DRWizards/DRFailoverWizardRecoverPage.zh-CN.resx:277]: This was also breaking the build.

**Review by danilo-delbusso — COMMENTED** [review]
Since you're also fixing the files, a few more issues with these two `resx` pages.

```xml
<data name="progressBar1.Anchor" type="System.Windows.Forms.AnchorStyles, System.Windows.Forms">
    <value>Top, Left, Right</value>
  </data>
```
This one doesn't exist in the base .resx for DRFailoverWizardRecoverPage.resx, it's only in the localized files


`buttonResolveAll.Size` in `DRFailoverWizardPRecheckPage` is different in the JA version ( [100,23] instead of [80,23])

I understand this might be related to the length of the text, but just checking


Similarly, `progressBar1.Location` is [3.71] in JA, and [3,378] in the others

**Review by danilo-delbusso — APPROVED** [review]

**Review by CitrixChris — APPROVED** [review]

**Pull request merged**